src/Xamarin.Android.Build.Tasks/Tasks/CheckGoogleSdkRequirements.cs

```diff
@@ -14,17 +14,26 @@ public class CheckGoogleSdkRequirements : AndroidTask
 	{
 		public override string TaskPrefix => "CGS";
 
-		[Required]
+		/// <summary>
+		/// This will be blank for .NET 5 builds
+		/// </summary>
 		public string TargetFrameworkVersion { get; set; }
 
+		/// <summary>
+		/// This is used instead of TargetFrameworkVersion for .NET 5 builds
+		/// </summary>
+		public int ApiLevel { get; set; }
```

Comment:
```suggestion
		public int TargetApiLevel { get; set; }
```

This is more clear in intent than the previous name.

Reply:
Should this instead be `string TargetAndroidId` (plus associated chicanery and conversions), so that it can “reasonably” support preview API levels, such as this year’s current `R` (and presumably next year’s `S`, and...)?

`AndroidManifest.xml` has historically allowed `//uses-sdk/@android:targetSdkVersion` to contain values such as `R`, and we’ve historically allowed `$(TargetFrameworkVersion)` to be e.g. `v10.0.99` to  use the preview API level, and we need *some* form of equivalent in .NET 5, especially since we’ll be *losing* `$(TargetFrameworkVersion)`...

Reply:
@jonp for .NET 5 the incoming value here will be `$(_AndroidApiLevel)`, which is currently coming from:

https://github.com/xamarin/xamarin-android/blob/3c94f6f9ae7902919620c1ca26d7ad77ff080620/src/Xamarin.Android.Build.Tasks/Tasks/ResolveAndroidTooling.cs#L188

I don't think we have a way to set this to an unstable API level yet in .NET 5, but the value would be set to `29` instead of `R`.

I think the name `ApiLevel` is OK, because we've used this for other tasks that take `$(_AndroidApiLevel)` as input:

https://github.com/xamarin/xamarin-android/blob/3c94f6f9ae7902919620c1ca26d7ad77ff080620/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets#L810

https://github.com/xamarin/xamarin-android/blob/3c94f6f9ae7902919620c1ca26d7ad77ff080620/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets#L1860

https://github.com/xamarin/xamarin-android/blob/3c94f6f9ae7902919620c1ca26d7ad77ff080620/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets#L2366

A couple older tasks use the name `AndroidSdkPlatform`, though.